src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.d.ts

```diff
@@ -5,5 +5,3 @@ declare module 'goog:optional.export.object.property' {
   import alias = ಠ_ಠ.clutz.optional.export.object.property;
   export = alias;
 }
-declare namespace ಠ_ಠ.clutz {
```

Comment:
what happened here?

Reply:
I moved the lines that created the namespace block inside the if statement.